reportportal/values.yaml

```diff
@@ -36,7 +36,9 @@ uat:
   sessionLiveTime: 86400
   podAnnotations: {}
   jvmArgs: "-Djava.security.egd=file:/dev/./urandom -XX:MinRAMPercentage=60.0 -XX:MaxRAMPercentage=90.0"
+  additionalEnvironments: {}
   securityContext: {}
+  nodeSelector: {}
```

Comment:
Please remove root `nodeSelector` values, since you update it in a way it becomes redundant.